pkg/apis/core/types.go

```diff
@@ -488,6 +496,17 @@ type PersistentVolumeClaimSpec struct {
 	// (Alpha) Using the namespace field of dataSourceRef requires the CrossNamespaceVolumeDataSource feature gate to be enabled.
 	// +optional
 	DataSourceRef *TypedObjectReference
+	// volumeAttributesClassName is the name of the VolumeAttributesClass required by the claim.
```

Comment:
Avoid word "required" here since this is optional?  Maybe "volumeAttributesClassName may be used to set the VolumeAttributesClass used by this claim"?